### ISSUE TITLE
Conflict with Biomes O' Plenty Hotspring water

### ISSUE THREAD

**Issue by InvalidArgument3** [connor135246/Simply-Hot-Springs#2]
Adding BoP together with this mod causes a crash when placing down a block of Simply Hot Springs' water, and trying to pick it up with a bucket. Also, BoP overrides the filled bucket added by this mod. 

[crash-2020-09-10_07.02.39-client.txt](https://github.com/connor135246/Simply-Hot-Springs/files/5201434/crash-2020-09-10_07.02.39-client.txt)

**Comment by connor135246** [comment]
Can't believe I never tested picking it up with a bucket. That part's an easy fix.

The problem is "Also, BoP overrides the filled bucket added by this mod." You see, forge allows multiple fluids to have the same name, but only one is allowed to be the default. No matter what type of hot spring water you pick up with your bucket, the one that gets dumped out is always BoP's.

I thought this would be as simple as making sure my fluid is registered before BoP's, so that mine would be the default for hot spring water instead of BoP's. Unfortunately, it would seem that BoP made the same registration mistake that I did - instead of crashing when I pick up my hot spring water, the game will crash if I try to pick up BoP's hot spring water. 

I could rename my hot spring water so that it's completely separate, but then any compatibility with hot spring water from another mod won't apply to my hot spring water. So I don't see any other option than to let BoP's hot spring water take priority.

tl;dr: crash fixed, but bop's water will still override this mod's water.

**Comment by connor135246** [comment]
Turns out there was a way to register this mod's hot spring water before BoP's and then basically brute force register BoP's immediately after.

So as of v0.2.0, this mod's hot spring water overrides BoP's.
